test/e2e/app-dir/dynamic-io/app/cases/static-rsc-dynamic-client/client.tsx

```diff
@@ -0,0 +1,9 @@
+'use client'
+
+export function Time() {
+  return (
+    <p>
+      The time is <span id="time">{new Date().toDateString()}</span>
```

Comment:
When you've frozen time in the test, you could also change this to an actual time.
```suggestion
      The time is <span id="time">{new Date().toISOString()}</span>
```
This would make the effect more obvious when running the app locally.